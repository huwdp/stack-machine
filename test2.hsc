push 60
push 9
add
prt
