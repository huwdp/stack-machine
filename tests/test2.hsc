pushi 60
pushi 9
addi
printl