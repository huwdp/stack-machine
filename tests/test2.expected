Welcome to Huw's Stack Machine
69
